.idea/misc.xml

```diff
@@ -1,5 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="EntryPointsManager">
```

Comment:
Well, this file exists in a hidden folder which is maintained by IDEA-based IDEs like IntelliJ. These stuffs were changed by Android Studio. I think it changes these files based on the system, version etc. and will change automatically.